### DISCUSSION TITLE
Refresh

### DISCUSSION THREAD

**Discussion by CokaCrack** [4gray/iptvnator#280] [Ideas]
When I refresh the playlist we lost everything we puts on favorite. 

Thanks for the great work by the way i will sure buy you a coffee.

EDIT: yeah the moment i wrote this i figure out i could just make a second playlist. one i keep for my favorite. one a can refresh for the ppv

**Comment by 4gray** [comment]
> Thanks for the great work by the way i will sure buy you a coffee.

Thank you for the support!
